src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).orElseThrow(()
+                -> new RuntimeException("Couldn't get driver by id " + id));
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/98731880/174574070-f0ebc5c9-5a68-4e24-8893-d403a69cc7e7.png)

Reply:
I disagree with that point.
If we change orElseThrow() to get() we'll get NPE in the fiture